ui/src/components/mobileservices/MobileServiceView.js

```diff
@@ -0,0 +1,81 @@
+import React, { Component } from 'react';
+import BoundServiceRow from './BoundServiceRow'
+import UnboundServiceRow from './UnboundServiceRow'
+class MobileServiceView extends Component {
+
+    constructor(props) {
+        super(props);
+    
+        this.state = {
+          boundServices: [{
+            serviceLogoUrl: 'https://pbs.twimg.com/profile_images/702119821979344897/oAC05cEB_400x400.png',
+            serviceName: 'Identity Management',
+            serviceId: 'dh-keycloak-apb-h7k9j',
+            serviceDescription: 'Identity Management - Identity and Access Management',
+            documentationUrl: 'http://docs.keycloak.org',
+            setupText: 'Identity Management SDK setup'
+          }, {
+            serviceLogoUrl: 'https://avatars1.githubusercontent.com/u/3380462?s=200&v=4',
+            serviceName: 'Mobile Metrics',
+            serviceId: 'dh-metrics-apb-wqm5c',
+            serviceDescription: 'Installs a metrics service based on Prometheus and Grafana',
+            setupText: 'Mobile Metrics SDK setups',
+            configuration : [{key:"A property",value:"A Value"},{key:"Another property",value:"Another Value"}]
+          }],
+          unboundServices: [{
+            serviceLogoUrl: 'https://pbs.twimg.com/profile_images/702119821979344897/oAC05cEB_400x400.png',
+            serviceName: 'Identity Management',
+            serviceId: 'dh-keycloak-apb-h7k9j',
+            serviceDescription: 'Identity Management - Identity and Access Management',
+            documentationUrl: 'http://docs.keycloak.org',
+            setupText: 'Identity Management SDK setup'
+          }, {
+            serviceLogoUrl: 'https://avatars1.githubusercontent.com/u/3380462?s=200&v=4',
+            serviceName: 'Mobile Metrics',
+            serviceId: 'dh-metrics-apb-wqm5c',
+            serviceDescription: 'Installs a metrics service based on Prometheus and Grafana',
+            setupText: 'Mobile Metrics SDK setups',
+            configuration : [{key:"A property",value:"A Value"},{key:"Another property",value:"Another Value"}]
+          }]
+        };
```

Comment:
@secondsun can you query the `<url of the service>/api/serviceinstances` endpoint to see if you have enough information to build this ui?

Reply:
Thanks, will do.

Is there a service object I can use?  If not do you mind if I add something that works like ScheduleService does here :  https://github.com/devnexus/devnexus-pwa/blob/master/script/Schedule.js#L13

Reply:
@secondsun you mean this [DataService](https://github.com/aerogear/mobile-client-service/blob/master/ui/src/DataService.js)?